src/guide/reviewing-and-sponsoring-a-change.md

```diff
@@ -0,0 +1,73 @@
+
+Sponsoring new contributions is an important activity - it's how the engineering culture of a project gets passed on from the core group to new Contributors, from one generation to the next. It should be fun, so please celebrate the contributions you've sponsored by mentioning them on your blog or similar. Congratulate other Sponsors on their work. Take pride in the value you provide to Contributors. Their success reflects well on you.
+
+As a Sponsor, Contributors will look up to you for guidance to get their contributions into the project - your actions will determine whether Contributors will feel welcome and want to engage further with the project beyond their initial attempt, or not. Let's not lose enthusiastic, engaged and technically competent Contributors due to a lack of communication. If there is a request in your area of expertise and you can't address it, at least acknowledge receipt of the request and provide an estimate for when you'll be able to give it your attention. A frank explanation of your time constraints or commitments will be appreciated and respected.
+
+There are many different reasons to sponsor a change and depending on the situation the exact steps involved may differ significantly. An experienced developer hired into a larger team of OpenJDK developers is likely to get much of the required information by osmosis and will likely need less help from a sponsor than the enthusiast sitting at home with no prior experience of large scale software development. This text aims to highlight the steps involved in sponsoring a change but can't cover all possible scenarios. As always, common sense takes precedence where the assumptions made here doesn't apply to your use case.
+
+## Reviewing a change on GitHub
+
+Code reviews in OpenJDK happens on [GitHub](https://github.com/openjdk). This guide will hint at the most common steps to perform a review but it's not a complete documentation of all features. For official documentation see the [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
+
+When you arrive at a pull request there are a few different sections that you can browse through, Conversation, Commits, Checks, and Files changed. We will look at the first and the last of these.
+
+### Conversation
+
+The Conversation section is where all discussion around the PR happens. Note that this section is intended for technical discussion around the actual implementation of a change. This is not the place to discuss the appropriateness of the change or have overarching design discussions, that should happen on the appropriate [mailing lists].
```

Comment:
I have to disagree about the last part - these days because the PR comments go to the mailing list and mailing-list-only comments (in a PR thread) go to the PR, it is indeed the case that design discussions and questions of appropriateness appear in the PR. Personally I prefer JBS to capture such things but the PR has become the one-place-shop in many cases and JBS commentary is typically very limited. Also JBS can't be used by a developer who needs a sponsor and doesn't yet have Author role.

Reply:
I think PRs are typically used to discuss the *implementation* once you have one. If someone opens a JBS issue with e.g. a new feature, but with no corresponding code, then the discussion will happen there. 

From time to time a PR is published with a concrete implementation, but then a discussion arises about the approriateness of the design, or even if the problem should be resolved at all etc. I don't think it is ideal if that kind of discussions happen on a PR, but maybe it's not that bad either. It is mirrored on the mailing list, and the only real problem in that case is that maybe the subject is bad -- it looks like discussions about a particular implementation, but is maybe indeed a discussion about the future direction on some functionality.

So, I dunno. Maybe we can recommend that people start up a new thread on the mailing list, if it turns out that some fundamental aspect of the PR needs to be discussed? I don't think it is a bad advice, but it could be formulated more softly as a suggestion.

Reply:
This is a good advice. I have updated the text to cover this.
I think we should encourage what we think is the right thing to do, not what people in general does - unless the two are the same of course :-)
There are many places in the guide that states that the mailing lists are the right place for discussions since that's what most OpenJDK developers think is the right thing to do.